src/utils/graphql/queries/getReviewInfo.query.ts

```diff
@@ -12,6 +12,7 @@ export default gql`
         status
         timeCreated
         level
+        isLastLevel
```

Comment:
Why this one needed to be returned here?

Reply:
https://github.com/openmsupply/application-manager-web-app/blob/4f7295b948dad54ce8560d01506182ef161107a3/src/utils/hooks/useGetFullReviewStructure.tsx#L81-L87

do you think I should change this to use reviewAssignment passed through from previous query ?

Reply:
I think this have been changed to get `isLastLevel` from review, so can be removed from reviewAssignment query?